webapp/src/components/modals/channel_settings/edit_channel_settings.tsx

```diff
@@ -277,6 +293,21 @@ export default class EditChannelSettings extends PureComponent<Props, State> {
             component = <Loading/>;
         }
 
+        const {showConfirmModal} = this.state;
+        const confirmComponent = (
+            <ConfirmModal
+                cancelButtonText={'Cancel'}
+                confirmButtonText={'Delete'}
+                confirmButtonClass={'btn btn-danger'}
+                hideCancel={false}
+                message={'Delete Subscription ?'}
```

Comment:
1/5 change to this
```
let confirmComponent;
if (this.props.selectedSubscription) {
    confirmComponent = (
        <Confirm Modal
            ...
            message={`Delete Subscription ${this.props.selectedSubscription.id`}} // will be sub.name later
        />
    );
}

Reply:
I like this and wasn't aware of the `s for obtaining a variable value in a print string